test-scripts/tomcat-deployment/infra-destroy.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+# This script is responsible for destroying the AWS instances and other AWS resources created
+
+stackName=TGdummy
+keyPairName=tgDummy
+
+echo "Infrastructure Destroy Initiated"
+
+# Deleting the created stack
+aws cloudformation delete-stack --stack-name $stackName
+
+# Deleting the created key pair
+aws ec2 delete-key-pair --key-name $keyPairName
+
+# TO-DO Hold the script till the instance is in terminated state
```

Comment:
Should this file end with a new line char? [1]

[1] https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
This is fixed with : 3b681e445a2e8d0047be71d8f83145453662dc71